Misc changes to Task Pages (#546)
huggingface/huggingface_hub: tasks/src/image-classification/about.md

## Use Cases

> Image classification models' use cases are cases in which we are not interested in specific instances of objects with location information or their shape. 

Reviewer: This sounded a bit strange, maybe:

```suggestion
Image classification models can be used when we are not interested in specific instances of objects with location information or their shape. 
```
